### ISSUE TITLE
valid-jsdoc rule fails for async function

### ISSUE THREAD

**Issue by marikaner** [buzinas/tslint-eslint-rules#375]
We have an async function that returns Promise<void>:

```ts
async function(): Promise<void> {
  await doSomething();
}
```

When this is annotated with `@returns` the rule shows me an error. Accoding to the es-lint documentation this is not the case. Can I work around that somehow?
